spot-client/src/spot-tv/ui/components/meeting-frame/meeting-frame.js

```diff
@@ -278,6 +279,11 @@ export class MeetingFrame extends React.Component {
         switch (type) {
         case COMMANDS.HANG_UP:
             this._jitsiApi.executeCommand('hangup');
+
+            if (data.force) {
```

Comment:
what is the `force` option needed for ? Maybe it would be worth putting a comment wherever that force argument is specified (not here but in the place where it's called).

Reply:
I used force to skip the feedback window. In the share flow, there was no talk of having feedback entry. So for now I skip feedback.

Reply:
Left some new comments. Let me know if there are more changes to be made. Maybe instead of force it should explicitly be skip feedback and renamed later if it does more than skipping feedback.

Reply:
Yeah, explicit 'skipFeedback' would be nice.